### PR TITLE
kubeprod: Fix build for Linuxbrew

### PR DESCRIPTION
**Pull request by sameersbn**
- [x] Have you followed the [guidelines for contributing](https://github.com/Linuxbrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Linuxbrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [x] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

### CODE REVIEW

**Comment by iMichka** [comment]
Hi. Version updates should go into https://github.com/Homebrew/homebrew-core. They are the merged back to this repo after some time and bottles will be rebuilt.

But I would be happy to keep the other changes though. Can you keep the ENV line and put an “if OS.mac?” after it. This helps me in case of a merge conflict.

**Comment by sameersbn** [comment]
@iMichka Would you recommend moving this contribution to https://github.com/Homebrew/homebrew-core instead. I ask this because in addition to the version update, the PR makes the recipe cross platform (macos/linux). Does that work for you?

**Comment by iMichka** [comment]
No, https://github.com/Homebrew/homebrew-core only knows about mac. There is not linux support in that part of the project. All the Linux changes are kept in this repository.

So the version update should go to https://github.com/Homebrew/homebrew-core, and the linux/mac split should stay here. The version update will come through a merge later on.

**Comment by sameersbn** [comment]
Thanks for the clarification @iMichka. I've reverted the version update in the PR and consolidated the install section.

**Comment by sameersbn** [comment]
is there anything I can do to fix the CI errors?

**Comment by iMichka** [comment]
You will need to rebase this on master. We had a bug in our CI.

Regarding the change: I would define the following variable:
`target = OS.mac? ? "darwin/amd64" : "linux/amd64"`

which you can then for both places where the change is needed.

**Comment by sameersbn** [comment]
updated! thanks.

File: Formula/kubeprod.rb [hunk 1]
```diff
@@ -16,13 +16,13 @@ class Kubeprod < Formula
 
   def install
     ENV["GOPATH"] = buildpath
-    ENV["TARGETS"] = "darwin/amd64"
+    ENV["TARGETS"] = OS.mac? ? "darwin/amd64" : "linux/amd64"
```
Comment (iMichka) [inline Formula/kubeprod.rb]:
```suggestion
    targets = OS.mac? ? "darwin/amd64" : "linux/amd64"
    ENV["TARGETS"] = targets
```

File: Formula/kubeprod.rb [hunk 2]
```diff
@@ -16,13 +16,13 @@ class Kubeprod < Formula
...
     dir = buildpath/"src/github.com/bitnami/kube-prod-runtime"
     dir.install buildpath.children
 
     cd dir do
       system "make", "-C", "kubeprod", "release", "VERSION=v#{version}"
-      bin.install "kubeprod/_dist/darwin-amd64/bkpr-v#{version}/kubeprod"
+      bin.install "kubeprod/_dist/" + ENV["TARGETS"].tr("/", "-") + "/bkpr-v#{version}/kubeprod"
```
Comment (iMichka) [inline Formula/kubeprod.rb]:
```suggestion
      bin.install "kubeprod/_dist/" + targets.tr("/", "-") + "/bkpr-v#{version}/kubeprod"
```

**Comment by iMichka** [comment]
Please remove the sha256 line for the linux bottle, and add `revision 1 unless OS.mac?`

**Comment by sameersbn** [comment]
all done!

**Comment by iMichka** [comment]
@sameersbn Thanks for your contribution!

**Pull request closed without merging**